### ISSUE TITLE
[Fixed] Error in log can't unmount

### ISSUE THREAD

**Issue by Stillhard** [canyie/Riru-IsolatedMagiskHider#2]
```
2021-01-24 18:45:06.310 1349-2486/? D/AMS: isProcStartable in LRU io.github.vvb2060.magiskdetector return true
2021-01-24 18:45:06.329 1349-2486/? I/ActivityManager: AMS: *** Start proc 4613:io.github.vvb2060.magiskdetector/u0i1 for service io.github.vvb2060.magiskdetector/.RemoteService
2021-01-24 18:45:06.359 4613-4613/? I/IsolatedMagiskHider: Created isolated process 4613, starting magisk hide...
2021-01-24 18:45:06.436 4613-4613/? D/IsolatedMagiskHider: hide_policy: Unmounted (/system/xbin)
2021-01-24 18:45:06.486 4613-4613/? D/IsolatedMagiskHider: hide_policy: Unmounted (/system/usr/share/zoneinfo)
2021-01-24 18:45:06.536 4613-4613/? D/IsolatedMagiskHider: hide_policy: Unmounted (/system/media/audio/ui)
2021-01-24 18:45:06.577 4613-4613/? D/IsolatedMagiskHider: hide_policy: Unmounted (/system/lib64)
2021-01-24 18:45:06.619 4613-4613/? D/IsolatedMagiskHider: hide_policy: Unmounted (/system/lib)
2021-01-24 18:45:06.688 4613-4613/? D/IsolatedMagiskHider: hide_policy: Unmounted (/system/framework)
2021-01-24 18:45:06.702 2392-2407/? D/GasService: FG app changed: from com.transsion.XOSLauncher to io.github.vvb2060.magiskdetector
2021-01-24 18:45:06.727 4613-4613/? D/IsolatedMagiskHider: hide_policy: Unmounted (/system/etc/permissions)
2021-01-24 18:45:06.767 4613-4613/? D/IsolatedMagiskHider: hide_policy: Unmounted (/system/bin)
2021-01-24 18:45:06.807 4613-4613/? D/IsolatedMagiskHider: hide_policy: Unmounted (/sbin)
2021-01-24 18:45:06.808 4613-4613/? E/IsolatedMagiskHider: hide_policy: can't unmount /system/etc/mkshrc: Permission denied
2021-01-24 18:45:06.808 4613-4613/? I/IsolatedMagiskHider: Unmounted magisk file system.
```
Is this a problem? @canyie

**Comment by canyie** [comment]
Try run this command:  ` magiskpolicy --live "allow zygote * filesystem { unmount }" `

**Comment by Stillhard** [comment]
> Try run this command: `magiskpolicy --live "allow zygote * filesystem { unmount }"`

Nice, that line solved it
Thx

**Comment by canyie** [comment]
Solved in 2864c0eae8d5d1ac84602329f8bd1ed34959323f
